modules/monitoring/files/usr/local/bin/govuk_uptime_collector

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env ruby
+
+# Usage: ./collect.rb <environment> <service ...>
+
+require "csv"
+require "fileutils"
+require "net/http"
+require "time"
+require "statsd"
+
+class Array
+  def each_after(n)
```

Comment:
You'd probably want to approach this with a [refinement](https://ruby-doc.org/core-2.1.1/doc/syntax/refinements_rdoc.html) rather than a monkey patch, I don't think it's actually necessary though